content/en/docs/kubernetes/collector/components.md

```diff
@@ -520,3 +522,142 @@ subjects:
     name: otel-collector-opentelemetry-collector
     namespace: default
 ```
+
+## Kubernetes Objects Receiver
+
+| Deployment Pattern   | Usable                                                   |
+| -------------------- | -------------------------------------------------------- |
+| DaemonSet (Agent)    | Yes, but will result in duplicate data                   |
+| Deployment (Gateway) | Yes, but more than one replica results in duplicate data |
+| Sidecar              | No                                                       |
+
+The Kubernetes Objects receiver collects, either by pulling or watching, objects
+from the Kubernetes API server. The most common use case for this receiver is
+watching Kubernetes events, but it can be used to collect any type of Kubernetes
+object. Since the receiver gathers telemetry for the cluster as a whole, only
+one instance of the receiver is needed across the cluster in order to collect
+all the data.
+
+Currently only a service account can be used for authentication. The service
+account also needs proper permissions to pull data from the Kubernetes API
+server (see below). If you're using the
+[OpenTelemetry Collector Helm chart](../../helm/collector/) and you want to
+ingest events, you can use the
+[`kubernetesEvents` preset](../../helm/collector/#cluster-metrics-preset) to get
+started.
+
+For objects configuring for pulling, the receiver will use the Kubernetes API to
+periodically list all the objects in the Cluster. Each object will be converted
+to its own log. For objects configured for watching, the receiver creates a a
+stream with the Kubernetes API and which receives updates as the objects change.
+
+To see which objects are available for collection run in your cluster run
+`kubectl api-resources`:
+
+<!-- cspell:disable -->
+
+```console
+❯ k api-resources
```

Comment:
```suggestion
k api-resources
```

Reply:
Reason why I suggest the chevron get removed is because people often copy-paste the whole line, but they'd need to remove that one part. Just a small point of friction.

Reply:
Made my own suggestion (bc I'm on my phone) to use kubectl also.